timesketch/frontend/src/components/Sigma/SigmaList.vue

```diff
@@ -15,104 +15,163 @@ limitations under the License.
+            :key="template.os">
+            {{ template.os }}
+          </option>
+        </b-select>
+        Parsed rule Search query:
+        <b><code>{{ parsed['search_query'] }}</code></b>
+        <explore-preview :searchQuery="parsed['search_query']">
+        </explore-preview>
+
+        <b-field label="Edit Sigma Rule" label-position="on-border"
+          style="margin-top: 25px;">
+          <b-input custom-class="ioc-input" type="textarea" rows="25"
+            v-model="editingRule.rule_yaml"
+            @input="parseSigma(editingRule.rule_yaml)"></b-input>
+        </b-field>
+        <b-field grouped>
+          <b-field grouped expanded position="is-right">
+            <p class="control">
+              <b-button type="is-primary" @click="addRule(editingRule)">
+                {{save_button_text}}
+              </b-button>
+            </p>
+            <p class="control">
+              <b-button @click="showEditModal = false">Cancel</b-button>
+            </p>
+          </b-field>
+        </b-field>
+
+      </section>
+    </b-modal>
+    <!-- End modal -->
+
+    <span class="icon is-small" style="cursor: pointer" title="Add a rule Rule"
+      @click="composeRule()"><i class="fas fa-plus"></i>Add
+    </span>
+
+    <b-table v-if="sigmaRuleList" :data="sigmaRuleList"
+      :current-page.sync="currentPage" :per-page="perPage" paginated
+      pagination-simple pagination-position="bottom"
+      default-sort-direction="desc" sort-icon="arrow-down"
+      sort-icon-size="is-small" icon-pack="fas" icon-prev="chevron-left"
+      icon-next="chevron-right" default-sort="title" key="props.row.id">
+
+      <b-table-column field="title" label="Name" v-slot="props" sortable
+        searchable>
+        <div @click="startRuleEdit(props.row)" custom-class="margintop-pointer">
           {{ props.row.title }}
         </div>
       </b-table-column>
 
-      <b-table-column field="ts_use_in_analyzer" label="Use in Analyzer" v-slot="props" sortable>
-        <div @click="props.toggleDetails(props.row)" style="margin-top:5px;cursor:pointer;">
-          {{ props.row.ts_use_in_analyzer }}
+      <b-table-column field="author" label="Author" v-slot="props" searchable
+        sortable>
+        <div @click="startRuleEdit(props.row)" custom-class="margintop-pointer">
+          {{ props.row.author }}
         </div>
-      </b-table-column>
 
-      <b-table-column field="description" label="Description" v-slot="props" searchable>
-        <div @click="props.toggleDetails(props.row)" style="margin-top:5px;cursor:pointer;">
-          {{ props.row.description }}
+      </b-table-column>
+      <b-table-column field="status" label="Status" v-slot="props" sortable
+        searchable>
+        <div @click="startRuleEdit(props.row)" custom-class="margintop-pointer">
+          {{ props.row.status }} <span class="icon is-small"
+            style="cursor: pointer"
```

Comment:
superfluous? maybe a class called "clickable" could be added here, and re-used in L113  and L116

Reply:
done